code/obj/flock/cage.dm

```diff
@@ -268,8 +269,9 @@
 	relaymove(mob/user as mob)
 		if (user.stat)
 			return
-
-		if(prob(25))
+		if(ON_COOLDOWN(src,"move_damage",1 SECONDS))
+			return
+		if(prob(75))
 			takeDamage("brute",1)
```

Comment:
```suggestion
		if(!ON_COOLDOWN(src,"move_damage",1 SECOND))
			if(prob(75))
				takeDamage("brute",1)
```
SECONDS -> SECONDS. Also, otherwise ignores first attack I think?